components/gcp/dataproc/delete_cluster/README.md

```diff
@@ -1,57 +1,80 @@
 
-# Dataproc - Delete Cluster
+# Deleting a Cluster with Cloud Dataproc
+A Kubeflow Pipeline component to delete a cluster in Cloud Dataproc service.
 
 ## Intended Use
-A Kubeflow Pipeline component to delete a cluster in Google Cloud Dataproc service. 
+Use the component to recycle a Dataproc cluster as one of the step in a KFP pipeline. This component is usually used with an [exit handler](https://github.com/kubeflow/pipelines/blob/master/samples/basic/exit_handler.py) to run at the end of a pipeline.
+
+## Runtime arguments
+Name | Description | Type | Optional | Default
+:--- | :---------- | :--- | :------- | :------
+project_id | The Google Cloud Platform (GCP) project ID that the cluster belongs to. | GCPProjectID | No |
+region | The Cloud Dataproc region runs the cluster to delete. | GCPRegion | No |
+name | The cluster name to delete. | String | No |
+wait_interval | The number of seconds to pause between polling the delete operation done status. | Integer | Yes | `30`
+
+## Cautions & requirements
+To use the component, you must:
+* Setup project by following the [guide](https://cloud.google.com/dataproc/docs/guides/setup-project).
+* The component is running under a secret of [Kubeflow user service account](https://www.kubeflow.org/docs/started/getting-started-gke/#gcp-service-accounts) in a Kubeflow cluster. For example:
+```
+component_op(...).apply(gcp.use_gcp_secret('user-gcp-sa'))
+```
+* Grant Kubeflow user service account the `roles/dataproc.editor` role on the project.
 
-## Run-Time Parameters:
-Name | Description
-:--- | :----------
-project_id | Required. The ID of the Google Cloud Platform project that the cluster belongs to.
-region | Required. The Cloud Dataproc region in which to handle the request.
-name | Required. The cluster name to delete.
-wait_interval | The wait seconds between polling the operation. Defaults to 30s.
+## Detailed Description
+This component deletes a Dataproc cluster by using [Dataproc delete cluster REST API](https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters/delete).
 
-## Sample
+Here are the steps to use the component in a pipeline:
+1. Install KFP SDK
+Install the SDK (Uncomment the code if the SDK is not installed before)
```

Comment:
Maybe delete 'Install the SDK'. It seems the words are duplicated.

Reply:
Done